src/app/app.component.ts

```diff
@@ -1,8 +1,14 @@
 import { Component } from '@angular/core';
+import { ToastService } from './core/service/toast.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css'],
```

Comment:
why did you remove this @aka-wtag bhai?

Reply:
@aka-wtag  bhai don't remove any css files or any other files which are generated by angular

Reply:
@aky-wtag vai resolved